docs/releases/2.2.txt

```diff
@@ -140,6 +140,8 @@ Database backends
 
 * Added result streaming for :meth:`.QuerySet.iterator` on SQLite.
 
+* Support for ``cx_Oracle`` < 6.0 is removed.
```

Comment:
Put under "Backwards incompatible" not "Minor Features" :-)

Reply:
:man_facepalming: